src/CachePlugin.php

```diff
@@ -379,7 +379,12 @@ private function createResponseFromCacheItem(CacheItemInterface $cacheItem)
 
         /** @var ResponseInterface $response */
         $response = $data['response'];
-        $response = $response->withBody($this->streamFactory->createStream($data['body']));
+        $stream = $this->streamFactory->createStream($data['body']);
+        if ($stream->isSeekable()) {
```

Comment:
If we should thrown an exception we could skip the `if ($stream->isSeekable())`. Calling `rewind` on a non-seekable stream will throw a RuntimeException according to PSR7.